python/hail/api2/matrixtable.py

```diff
@@ -1494,7 +1494,7 @@ def explode_cols(self, field_expr):
             elif self._fields[field_expr].indices != self._col_indices:
                 raise ExpressionException("Method 'explode_cols' expects a field indexed by col, found axes '{}'"
                                           .format(self._fields[field_expr].indices.axes))
-            s = 'sa.`{}`'.format(field_expr)
+            s = 'sa.{}'.format(escape_id(field_expr))
```

Comment:
I think you are missing `aggregate_*`.
Does the `key_expr` in `group_rows_by` and `group_cols_by` need to be escaped?

Reply:
> I think you are missing aggregate_* 
Where? The aggregate_rows, cols, entries methods handle the names in Python.

> does the key_expr need to be escaped
nope, mostly because we don't let you name it right now!